src/components/app/style.scss

```diff
@@ -1,3 +1,58 @@
-.app h1 {
-  color: green;
+.app {
```

Comment:
Slap a comment like "Temp styles. These will be reviewed before going live" at the top & I'm happy for this to merge.